products/sle/main.pm

```diff
@@ -645,6 +645,7 @@ sub load_inst_tests {
         if (    !consolestep_is_applicable()
             and !get_var("REMOTE_CONTROLLER")
             and !is_hyperv_in_gui
+            and !is_bridged_networking
```

Comment:
I don't mind disabling but without a comment here we will have a hard time understanding why it's disabled.

Reply:
*points @okurz to the comment 4 lines above*

+        # The test also won't work reliably when network is bridged (non-s390x svirt).

Reply:
sure, this is what @mnowaksuse added now. With this the question is resolved.